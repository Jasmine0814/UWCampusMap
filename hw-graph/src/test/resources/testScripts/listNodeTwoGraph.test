# test many nodes in two graphs can be listed correctly.

CreateGraph A
CreateGraph B
AddNode A n1
AddNode A n2
AddNode A n3
AddNode B n4
AddNode B n5
AddNode B n6
ListNodes A
ListNodes B

