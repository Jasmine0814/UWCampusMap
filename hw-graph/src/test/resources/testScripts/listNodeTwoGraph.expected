# test many nodes in two graphs can be listed correctly.

CreateGraph A
CreateGraph B
added node n1 to A
added node n2 to A
added node n3 to A
added node n4 to B
added node n5 to B
added node n6 to B
A contains: n1 n2 n3
B contains: n4 n5 n6

